File: php/aulas/2024/1_Semestre_PHP_Banco/aula03.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <?php

    /*
    ESTRUTURAS DE CONTROLE:
    - Estrutura Sequêncial
    - Estrutura de condição
    - Estrutura de repetição
    */

    /*
    ESTRUTURA DE CONDIÇÃO
    - if
    - if...else
    */

    $numero = 5;
    if($numero>0){
        echo "<p> Número Positivo </p>";
    }

    /*
    Se no bloco da estrutura possuir
    apenas uma instrução, as chaves
    são opcionais
    */

    if($numero>0)
        echo "<p> Número positivo </p>";
    

    if($numero>=0){
        echo "<p> Número positivo </p>";
    }else{
        echo "<p> Número negativo </p>";
    }

    /*
    Utilizando a estrutura if...else, 
    armazene um valor inteiro em uma
    variável. Logo em seguida, imprima
    na página em título de nível 2 se
    o número é POSITIVO, NEGATIVO ou NULO. 
    */

    // POSSIBILIDADE 1
    $num = 15;
    if($num>0){
        echo "<h2>Positivo</h2>";
    }
    if($num<0){
        echo "<h2>Negativo</h2>";
    }
    if($num==0){
        echo "<h2>Nulo</h2>";
    }

    // Possibilidade 2
    if($num>0){
        echo "<h2>Positivo</h2";
    }else{
        if($num<0){
            echo "<h2>Negativo</h2>";
        }else{
            if($num==0){
                echo "<h2>Nulo</h2>";
            }
        }
    }

    // Possibilidade 3
    if($num>0)
        echo "<h2>Positivo</h2";
    else if($num<0)
            echo "<h2>Negativo</h2>";
        else
                echo "<h2>Nulo</h2>";
    ?>
</body>
</html>

File: python/Aulas/2026_integrado/python/aula_03.py
'''
ESTRUTURAS
- Sequencial
- Condicional
- Repetição
'''

'''
ESTRUTURA CONDICIONAL
- Estrutura que muda o fluxo
de execução do algoritmo de acordo
com a avaliação de uma condição

if
if...else
if...elif...else
'''
'''
Salve um valor inteiro em uma variável,
posteriormente imprima o texto "É ZERO"
se o valor armazenado for 0.
'''
numero = 5

if numero == 0:
    print("É ZERO")


'''
Salve um valor inteiro em uma variável,
posteriormente imprima o texto "É ZERO"
se o valor armazenado for 0, ou imprima
"NÃO É ZERO" se o valor for diferente de 0.
'''
# 1º OPÇÃO 
if numero == 0:
    print("É ZERO")

if numero != 0:
    print("NÃO É ZERO")

# 2º OPÇÃO:
if numero == 0:
    print("É ZERO")
else:
    print("NÃO É ZERO")


'''
ARMAZENE UM NÚMERO INTEIRO EM UMA VARIÁVEL.
AO FINAL, IMPRIMIA "ZERO" SE O NÚMERO FOR 0,
"POSITIVO" SE O NÚMERO FOR POSITIVO, OU "NEGATIVO"
SE O NÚMERO FOR NEGATIVO.
'''
# 1º OPÇÃO
if numero == 0:
    print("zero")

if numero > 0:
    print("POSITIVO")

if numero < 0:
    print("NEGATIVO")

# 2 OPÇÃO
if numero==0:
    print("ZERO")
else:
    if numero > 0:
        print("POSITIVO")
    else:
        print("NEGATIVO")

# 3 OPÇÃO
if numero==0:
    print("ZERO")
elif numero > 0:
    print("POSITIVO")
else:
    print("NEGATIVO")
